specs/registry/perf-issue.json

```diff
@@ -43,7 +43,7 @@
       "description": "Reflection is being used by Compare/Equality functions. This could be because of a missing `Equals` override or incorrect `IEquatable<T>`/`IEqualityComparer<T>` implementation.",
       "docURL": null,
       "recommendation": "This can be fixed by providing the missing `Equals` override or properly implementing the `IEquatable<T>`/`IEqualityComparer<T>` interface.",
-      "rationale": "{value}% of your {issueCategory} was spent in `{function}`, We expected this value to be {relation} {criteria}%.",
+      "rationale": "{value}% of your {issueCategory} was spent in `{function}`. We expected this value to be {relation}&#8203;{criteria}%.",
```

Comment:
What is Unicode of &#8203; stands for? Why do we need it instead of the plain whitespace?